packages/lab/src/list-next/ListItemNext.tsx

```diff
@@ -0,0 +1,82 @@
+import { forwardRef, HTMLAttributes, MouseEvent } from "react";
+import { clsx } from "clsx";
+import { makePrefixer, useId } from "@salt-ds/core";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import listItemNextCss from "./ListItemNext.css";
+import { useListItems } from "./ListNextContext";
+
+const withBaseName = makePrefixer("saltListItemNext");
+
+export interface ListItemNextProps extends HTMLAttributes<HTMLLIElement> {
+  label?: string;
+  /**
+   * If true, the particular list item in list will be disabled.
+   */
+  disabled?: boolean;
+  selected?: boolean;
+  id?: string;
+  value: string;
+}
+
+export const ListItemNext = forwardRef<HTMLLIElement, ListItemNextProps>(
+  function ListItemNext(
+    {
+      children,
+      className,
+      disabled: disabledProp,
+      selected: selectedProp,
+      label,
+      id: idProp,
+      value,
+      onClick,
+      ...props
+    },
+    ref
+  ) {
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-list-item-next",
+      css: listItemNextCss,
+      window: targetWindow,
+    });
+
+    const listContext = useListItems();
```

Comment:
I normally prefer to destructure values here. Makes it easier for me to see what it being used

Reply:
given `id` and `disabled` will be declared below and would have to be renamed, and we need to check that the context is not coming with the default undefined, readability wise it ends up being a bit of the same imo, but happy to destructure it.